sources/packages/api/src/cra-integration/cra-integration.service.ts

```diff
@@ -167,38 +163,30 @@ export class CRAIntegrationService {
     }
   }
 
-  private createFooter(
-    code: TransactionCodes,
-    processDate: Date,
-    sequence: number,
-    recordCount: number,
-  ): CRAFileFooter {
-    const footer = new CRAFileFooter();
-    footer.transactionCode = code;
-    footer.processDate = processDate;
-    footer.programAreaCode = this.craConfig.programAreaCode;
-    footer.environmentCode = this.craConfig.environmentCode;
-    footer.sequence = sequence;
-    footer.recordCount = recordCount + 2; // Must be the number of records + header + footer.
-    return footer;
-  }
-
   /**
    * Expected file name of the CRA request file.
    * @param sequence file sequence number.
    * @returns Full file path of the file to be saved on the SFTP.
    */
-  createRequestFileName(sequence: number): string {
+  createRequestFileName(sequence: number): {
+    fileName: string;
+    filePath: string;
+  } {
     const sequenceFile = sequence.toString().padStart(5, "0");
-    return `${this.craConfig.ftpRequestFolder}\\CCRA_REQUEST_${this.craConfig.environmentCode}${sequenceFile}.DAT`;
+    const fileName = `CCRA_REQUEST_${this.craConfig.environmentCode}${sequenceFile}.DAT`;
+    const filePath = `${this.craConfig.ftpRequestFolder}\\${fileName}`;
```

Comment:
@andrewsignori-aot , i did get `//` in the file path? could you explain

Reply:
I am not sure if I understood but, if the question is about the "\\", it actually represents the regular "\" but it needs to be escaped with an additional "\" because the "\" is also the scape character.

Reply:
typo , yes //